devito/ops/transformer.py

```diff
@@ -1,3 +1,38 @@
+from devito.ir.iet import Expression, FindNodes
+from devito.ops.node_factory import OPSNodeFactory
+from devito.ops.nodes import OPSKernel
+from devito.ops.utils import namespace
+
+
+def opsit(trees, count):
+    node_factory = OPSNodeFactory()
+    expressions = []
+    for tree in trees:
+        expressions.extend([
+            Expression(make_ops_ast(i.expr, node_factory))
+            for i in FindNodes(Expression).visit(tree.inner)
+        ])
+
+    arguments = set()
+    to_remove = []
+    for exp in expressions:
+        func = [f for f in exp.functions if f.name != "OPS_ACC_size"]
+        arguments |= set(func)
+        if exp.is_scalar_assign:
+            to_remove.append(exp.write)
+
+    arguments -= set(to_remove)
+
+    callable_kernel = OPSKernel(
+        namespace['ops_kernel'](count),
+        expressions,
+        "void",
+        list(arguments)
+    )
+
+    return callable_kernel, None
```

Comment:
I don't really see the use of returning the `None`. 
I mean if not needed now, just return the first argument. In the future, when the PR for `par_loop_block` is created then this will be added.